src/components/templates/Header.jsx

```diff
@@ -93,16 +95,16 @@ const Header = () => {
               icon="/images/phoneReciever.svg"
               text="1005-346-272"
               textStyle="text-black"
-              styles="w-max px-2"
+              styles="w-max px-8"
             />
             <Button
```

Comment:
You should apply the base style for the button in the button component itself, with this we don't need to add classes every time, and later on, if we need any other variations, we will create those based on the conditions.